root/instrument/edit_form.tt

```diff
@@ -14,6 +14,15 @@
           [%- r.textarea('description', { rows => 5 }) -%]
           [%- field_errors(form, 'description') -%]
       [%- END -%]
+      [% WRAPPER form_row %]
+        <p>
+          [% l('HTML tags allowed in the description: {tag_list}.', 
+            {tag_list => comma_only_list(
+              ['a', 'abbr', 'br', 'code', 'em', 'li', 'p', 'span', 'strong', 'ul'],
```

Comment:
Please define this list along with `htmlTagName` in `root/static/scripts/common/i18n/expand2react.js`.

Reply:
This can't really be done just yet since this is a TT file, but I added a comment here to move it to expand2react once this is converted.